lib/path.js

```diff
@@ -210,6 +221,9 @@ win32.join = function() {
 // to = 'C:\\orandea\\impl\\bbb'
 // The output of the function should be: '..\\..\\impl\\bbb'
 win32.relative = function(from, to) {
+  assertPath(from);
+  assertPath(to);
```

Comment:
maybe this should be changed to be like [line #328](https://github.com/iojs/io.js/pull/1153/files#diff-c0fae59301ef49fb71b43278db9ad881R328)?